packages/blocks/src/routes/+page.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang='ts'>
 
-import { SlamMap2D } from '$lib';
 import pointcloudUrl from '$lib/assets/pointcloud.pcd?url'
 import motionPath from '$lib/assets/cbirrtsmooth800.txt?raw'
+import { MapLibre, SlamMap2D } from '$lib';
+import NavigationMap from './navigation-map.svelte';
 
 const fetchPointcloud = async () => {
   const response = await fetch(pointcloudUrl)
```

Comment:
Do we need non-2xx response checking for this request?

Reply:
This is just playground :P this request will never see a prod env